aten/src/ATen/native/native_functions.yaml

```diff
@@ -3725,7 +3726,7 @@
     CPU, CUDA: relu
     MkldnnCPU: mkldnn_relu
     QuantizedCPU: relu_quantized_cpu
-    NestedTensor: NestedTensor_relu
+    NestedTensorCPU, NestedTensorCUDA: NestedTensor_relu
```

Comment:
I'm pretty sure that at least some of these NestedTensor kernels handle the cpu vs. cuda case by using `DISPATCH_STUB()`, which does an extra dynamic dispatch to the corresponding cpu vs. cuda handler.

Depending on how perf-sensitive all of these nested tensor kernel are, you could rewrite them to remove the `DISPATCH_STUB` call and just write separate kernels here for CPU + CUDA.

(tbh getting rid of the DISPATCH_STUB usage is probably the right thing to do anyway, although it also comes with a nice minor perf improvement)

Reply:
> 'm pretty sure that at least some of these NestedTensor kernels handle the cpu vs. cuda case by using DISPATCH_STUB()

I wrote them and I don't know what DISPATCH_STUB is, so I don't think they use it.

Reply:
oh nvm then (I thought it might apply to some of the pre-existing NestedTensors before you added more)